tests/test_compute_utils.py

```diff
@@ -95,30 +96,35 @@ def test_surface_integrals(self):
         should also be done on grids with duplicate nodes (e.g. endpoint=True).
         """
 
-        def test(surface_label, grid):
-            q = np.random.random_sample(size=grid.num_nodes)
+        def test_B_theta(surface_label, grid, eq):
+            q = eq.compute("B_theta", grid=grid)["B_theta"]
```

Comment:
I had changed this earlier to confirm B_theta was computed correctly before we fixed that plotting issue. It's better than a random quantity test, so I kept the change.